app/jenkins-for-jira-ui/src/App.tsx

```diff
@@ -49,17 +52,26 @@ export const environmentSettings = {
 
 const AppContainer = styled.div`
 	color: #172B4D;
-	margin: 24px 24px 24px 0;
+	margin: 24px 36px 24px 0;
```

Comment:
Had to shift this a little as it was causing the content to be a little off-centre. Existing still looks ok.

<img width="1501" alt="Screenshot 2023-11-16 at 12 00 55 pm" src="https://github.com/atlassian/jenkins-for-jira/assets/37155488/c25f5b8f-9e5f-4fe9-849a-7dd2f2ba6b08">

![Uploading Screenshot 2023-11-16 at 12.02.44 pm.png…]()